images/shellcheck/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
```

Comment:
This file should be under `test/images/shellcheck`, to keep the test code separate from the "production" code.

Reply:
thanks. fixed.